bridges/scalajs-1.0/src/main/scala/bloop/scalajs/JsBridge.scala

```diff
@@ -1,17 +1,21 @@
 package bloop.scalajs
 
-import java.nio.file.Path
+import java.nio.file.{Files, Path}
 
 import bloop.config.Config.{JsConfig, LinkerMode, ModuleKindJS}
 import bloop.data.Project
 import bloop.logging.{DebugFilter, Logger => BloopLogger}
 import bloop.scalajs.jsenv.{JsDomNodeJsEnv, NodeJSConfig, NodeJSEnv}
-import org.scalajs.io.{AtomicWritableFileVirtualBinaryFile, MemVirtualBinaryFile}
-import org.scalajs.jsenv.Input
-import org.scalajs.linker.irio.{FileScalaJSIRContainer, IRFileCache}
-import org.scalajs.linker.{LinkerOutput, ModuleInitializer, ModuleKind, Semantics, StandardLinker}
 import org.scalajs.logging.{Level, Logger => JsLogger}
-import org.scalajs.testadapter.TestAdapter
+import org.scalajs.linker.{PathIRContainer, PathIRFile, PathOutputFile, StandardImpl}
+import org.scalajs.linker.interface.{ModuleKind => ScalaJSModuleKind, _}
+import org.scalajs.jsenv.Input
+import org.scalajs.testing.adapter.{TestAdapter, TestAdapterInitializer}
+import scala.collection.JavaConverters._
+
+import scala.concurrent.{Await, Future}
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
I don't think we should use the global scheduler here, bloop should be managing this thread pool itself. Can we pass either a scheduler or an execution context through the bloop <> scala.js boundaries?
  
Also, I'm curious, why is the scheduler needed now and it wasn't before? I see the Scala.js API now returns some  `Future`s.

Reply:
Yes, it should be possible to pass an `ExecutionContext` to this function. I will change it.

Note that `BloopComRun` also uses the global `ExecutionContext`, but it would be better to fix it separately.